src/main/java/frc/robot/controls/Driver.java

```diff
@@ -83,8 +96,20 @@ public double getSteer()
         double steerValue =  steer.getX();
         if (Math.abs(steerValue) < DEADZONE) return 0;
 
-        steerValue *= MAX_STEER_SPEED;
+        steerValue *= steerSpeed;
 
         return steerValue;
     }
+
+    public void useLowSpeed() {
+        throttleSpeed = 0.3;
+        steerSpeed = 0.2;
```

Comment:
we could define these as constants up top so that it's easy to change during competition if needed.

Reply:
I'll do that.